x-pack/solutions/security/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/insights/use_trigger_scan.ts

```diff
@@ -28,26 +26,67 @@ interface UseTriggerScanConfig {
 
 export const useTriggerScan = ({ onSuccess }: UseTriggerScanConfig) => {
   const { http } = useKibana().services;
+  const defendInsightsPolicyResponseFailureEnabled = useIsExperimentalFeatureEnabled(
+    'defendInsightsPolicyResponseFailure'
+  );
   const toasts = useToasts();
 
   const { data: anonymizationFields } = useFetchAnonymizationFields();
 
-  return useMutation<DefendInsightsResponse, { body?: { error: string } }, UseTriggerScanPayload>(
-    ({ endpointId, connectorId, actionTypeId }: UseTriggerScanPayload) =>
-      http.post<DefendInsightsResponse>(DEFEND_INSIGHTS, {
-        version: API_VERSIONS.internal.v1,
-        body: JSON.stringify({
-          endpointIds: [endpointId],
-          insightType: DefendInsightTypeEnum.incompatible_antivirus,
-          anonymizationFields: anonymizationFields.data,
-          replacements: {},
-          subAction: 'invokeAI',
-          apiConfig: {
-            connectorId,
-            actionTypeId,
-          },
-        }),
-      }),
+  return useMutation<DefendInsightsResponse[], { body?: { error: string } }, UseTriggerScanPayload>(
+    async ({ endpointId, connectorId, actionTypeId, insightTypes }: UseTriggerScanPayload) => {
+      const enabledTypes = insightTypes.filter((type) => {
+        if (type === 'policy_response_failure' && !defendInsightsPolicyResponseFailureEnabled) {
```

Comment:
Since we're already checking the FF before passing in `insightTypes`, is this check necessary?